experimental/traffic-portal/src/app/api/server.service.ts

```diff
@@ -186,4 +187,33 @@ export class ServerService extends APIService {
 
 		return this.put(`servers/${id}/status`, {offlineReason, status}).toPromise();
 	}
+
+	/**
+	 * Creating new Status.
+	 *
+	 * @param data containes name and description for the status.
+	 * @returns The 'response' property of the TO status response. See TO API docs.
+	 */
+	public async createStatus(data: ResponseStatus): Promise<ResponseStatus> {
```

Comment:
this should not take in a `ResponseStatus` parameter; response versions of things have read-only fields generated by the server like `id` and `lastUpdated`, and have different field typings regarding null vs undefined vs "zero-valued" than the objects TO requires in requests. It should use a `RequestStatus` instead.